OrbitCore/Utils.h

```diff
@@ -74,8 +74,8 @@ inline std::string GetEnvVar(const char* a_Var) {
   return var;
 }
 
-inline unsigned long long StringHash(const std::string& a_String) {
-  return XXH64(a_String.data(), a_String.size(), 0xBADDCAFEDEAD10CC);
+inline uint64_t StringHash(const std::string& string) {
```

Comment:
Can/should const std::string& be a string_view?

Reply:
Yes it can, I changed it, thanks.